packages/babel-plugin-transform-es2015-classes/src/vanilla.js

```diff
@@ -22,6 +22,14 @@ const noMethodVisitor = {
   },
 };
 
+const transformToObjectDefineProperty = template(`
+  Object.defineProperty(constructor, prop, {
```

Comment:
So if you want to replace something in a template you need it to be all caps

Reply:
Thanks will make this change and will keep this in mind. :)